features/desktop/notifications.feature

```diff
@@ -101,6 +101,26 @@ Feature: Notifications
     Then I should see "mentioned you in the post"
     And I should have 1 email delivery
 
+  Scenario: someone mentioned me in a comment
+    Given a user with email "bob@bob.bob" is connected with "alice@alice.alice"
+    And "alice@alice.alice" has a public post with text "check this out!"
+    And "bob@bob.bob" has commented mentioning "alice@alice.alice" on "check this out!"
+    When I sign in as "alice@alice.alice"
+    And I follow "Notifications" in the header
+    Then the notification dropdown should be visible
+    Then I should see "mentioned you in the comment"
+    And I should have 1 email delivery
+
+  Scenario: I mark a mention in comment notification as read
```

Comment:
> Because it doesn't matter. The "mark as read/unread" feature doesn't depend on the notification type. So it is OK to have a scenario that tests it in the frontend, but I don't want cucumber tests for every notification type to test if you can mark it as read. (that is something you can do in unit-test, which are faster).

I disagree. IMO, ideally, we should have an integration tests covering each client behaviour scenario. We can't rely on unit tests here. Unit tests are not for prooving that our application works as expected in every usage scenario. Ideally, we should have an integration test for every possible situation that may happen from the user point of view. The execution flow for "mark as read" for mentions in comments and posts is the same, but that is an implementation detail. We can't use that knowledge to claim that 
> "mark as read/unread" feature doesn't depend on the notification type.

In the end, I'll implement what you say, but I don't feel it right to have only one test scenario for "mark as read" for mentioning.